src/types/models/carSales.ts

```diff
@@ -0,0 +1,25 @@
+export interface Buyer {
+  email: string
+  firstName: string
+  lastName: string
+  phone: string
+}
+
+export interface ListingQuery {
+  externalListingId: string
+  firstRegistrationDate: string
+  id: number
+  image: string
+  make: string
+  mileage: number
+  model: string
+  price: number
+  type: string
+}
+
+export interface DealerCarSales {
```

Comment:
Why the "Dealer" prefix?

Reply:
I was thinking about the Dealer. But makes sense `CarSales`